website-1.x/data/users.js

```diff
@@ -506,14 +506,14 @@ module.exports = [
   },
   {
     caption: 'pnpm',
-    image: 'https://pnpm.js.org/img/pnpm.svg',
+    image: '/img/users/pnpm.svg',
     infoLink: 'https://pnpm.js.org/',
     fbOpenSource: false,
     pinned: false,
   },
   {
     caption: 'Polymath Network',
-    image: 'https://developers.polymath.network/img/text.svg',
+    image: '/img/users/polymatch-logo.png',
```

Comment:
how comes it's not a png?

Reply:
I took png image from official website https://polymath.network/